tmt/utils.py

```diff
@@ -1066,6 +1066,15 @@ def workdir(self) -> Optional[Path]:
 
         return self._workdir
 
+    @property
+    def clone_dirpath(self) -> Path:
+        """
+        Path for cloning into
+        Used internally for picking specific libraries (or anything else) from cloned repos
+        for filtering purposes, it is removed at the end of relevant step
+        """
+        return Path('/tmp/tmtclonedir')
```

Comment:
Is the hardcoded value a temporary one? An envvar to allow modifications would be welcome.